pandas/tests/io/test_html.py

```diff
@@ -122,6 +122,7 @@ def test_to_html_compat(self):
         tm.assert_frame_equal(res, df)
 
     @tm.network
+    @pytest.mark.xfail(reason="connection issues xref 34893")
     def test_banklist_url_positional_match(self):
         url = "http://www.fdic.gov/bank/individual/failed/banklist.html"
         # Passing match argument as positional should cause a FutureWarning.
```

Comment:
We actually use `self.banklist_data` (an offline version of this html) in other tests. 
But I am not sure if there was a reason that those two tests were still using an actual url? (doesn't seem to be any reason for that)

Reply:
yeah not really sure why these are failing, can figure out later, just trying to get CI green:->

Reply:
the data is back somehow xD

https://www.fdic.gov/resources/resolutions/bank-failures/failed-bank-list/banklist.html

Reply:
ok, must have been a blip.